src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -498,6 +498,12 @@ StDebugger >> debuggerInspectorModelClass [
 	^ StDebuggerInspectorModel
 ]
 
+{ #category : 'initialization' }
+StDebugger >> debuggerSettings [
```

Comment:
the method name should be simply "settings"

Reply:
it should also be class side

Reply:
You mean?

```smalltalk
initializeWindow: aWindowPresenter

 "blabla"
self class settings initializeSettings: aWindowPresenter.

```

Reply:
Rather something like `self setting setWindowPosition: aWindowPresenter` and `self settings`is implemented as `^self class settings`.

The parameter to a method named `initializeSettings:` should not be a presenter but a setting or something else.

Reply:
Done